CHANGELOG.md

```diff
@@ -13,7 +13,9 @@
 - Migrate `BloodPressureHistoryScreen` to a fragment
 
 ### Features
-- [In Progress: 06 Jul 2021] Add support for finding a patient online from ID scan
+- [In Progress: 06 Jul 2021]
+  - Add support for finding a patient online from ID scan
+  - Delete patients and it's complete medical records when the retention time has passed
```

Comment:
We can just leave the previous changelog "Add support for finding a patient online from ID scan". As far as the users are aware, they don't care about retention times and automatic removal. We can move that change to internal.